docs/docs/adding-forms.md

```diff
@@ -1,35 +1,35 @@
 ---
-title: Adding Forms
+title: फॉर्म को जोड़ना
 ---
 
-Gatsby is built on top of React. So anything that is possible with a React form is possible in Gatsby. Additional details about how to create React forms can be found in the [React forms documentation](https://reactjs.org/docs/forms.html) (which happens to be built with Gatsby!)
+Gatsby के ऊपर बनाया गया है React. तो सब कुछ जो संभव है React फॉर्म में ,वो संभव है Gatsby में भी. अतिरिक्त जानकारि की कैसे बनाएं React फॉर्म ,यहाँ पाया जा सकता है [React forms documentation](https://reactjs.org/docs/forms.html) (जो बनाया गया है Gatsby! से)
 
-Start with the following page.
+सेुरू करें निम्नलिखित पृष्ठ से.
```

Comment:
`सेुरू करें निम्नलिखित पृष्ठ से.` => `निम्नलिखित पेज से सुरु करें।`

Reply:
add a line break on line 6

Reply:
you have to add line break at line 6. you have added it at line 7.

add the line break [here](https://github.com/gatsbyjs/gatsby-hi/pull/59/files#diff-4761724f02e34e9e96f5808a7f4adfa9R6)

Reply:
> you have to add line break at line 6. you have added it at line 7.
> 
> add the line break [here](https://github.com/gatsbyjs/gatsby-hi/pull/59/files#diff-4761724f02e34e9e96f5808a7f4adfa9R6)

This is still not fixed